lib/@empirica/core/src/admin/classic/classic.ts

```diff
@@ -227,6 +241,12 @@ export function Classic({
       player.participantID = participantID;
       playersForParticipant.set(participantID, player);
 
+      player.updateConnectivityInfo({
+        isConnected: true,
+        connectedAt: new Date().toISOString(),
+        disconnectedAt: null
+      })
+
```

Comment:
Right here, we don't know if the participant is online. That's checked at the end of this function `if (online.has) {`.